src/components/SearchMessages.vue

```diff
@@ -30,13 +30,12 @@
 					{{ t("mail", "Search parameters") }}
 				</NcActionButton>
 			</NcActions>
-			<NcModal v-if="moreSearchActions"
+			<NcDialog v-if="moreSearchActions"
```

Comment:
this component still has styles for `.search-modal .modal-container {` and deeper. are they still used? if not -> :fire: